### DISCUSSION TITLE
[Vite] How to migrate serverDependenciesToBundle?

### DISCUSSION THREAD

**Discussion by jansedlon** [remix-run/remix#7872] [Q&A]
Hi 👋 I'd like to ask how to migrate the existing `serverDependenciesToBundle` property? There's a few packages like `remix-i18next`, `accept-language-parser`, etc.. which I had to enter. 

I couldn't find a similar option with Vite. Sadly, If I don't have this option, importing these libraries causes errors. In case of `remix-i18next`, when i try to import it, it throws an error like `Unexpected token "export"` inside `remix-i18next`.

I've got `"type": "module"` in my `package.json` and also `"module": "NodeNext", "moduleResolution":"nodenext", "allowImportingTsExtensions": true` in my `tsconfig.json`

**Comment by tylergraf** [comment]
I'm experiencing the same issue. We were using `serverDependenciesToBundle` for our shared components pulled in through `node_modules`. Each of those pull in their own `css` file.

**Comment by hi-ogawa** [comment]
It's hard to judge without more details and reproduction but this issue https://github.com/remix-run/remix/issues/7865 might be related and I wrote potential workaround there.

**Reply by tylergraf** [reply]
This seems to have worked for me. Thank you!

**Comment by AdityaSaroj** [comment]
Hey, did you find a solution for this?

**Comment by andorthehood** [comment]
For anyone who ends up here after searching for the vite config equivalent of `serverDependenciesToBundle`: you can use the `ssr.noexternal` option: https://vitejs.dev/config/ssr-options.html#ssr-noexternal
